media-libs/opencollada/opencollada-1.6.23.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/KhronosGroup/OpenCOLLADA.git"
+inherit multilib cmake-utils
+[[ ${PV} == "9999" ]] && inherit git-r3
+
+DESCRIPTION="Stream based read/write library for COLLADA files"
+HOMEPAGE="http://www.opencollada.org/"
+[[ ${PV} == "9999" ]] || \
+	SRC_URI="https://github.com/KhronosGroup/OpenCOLLADA/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+[[ ${PV} == "9999" ]] || \
+	KEYWORDS="~amd64 ~ppc64 ~x86" \
+	S="${WORKDIR}"/OpenCOLLADA-${PV}
+
+IUSE="expat static-libs"
+
+# This is still needed to have so version numbers
+MY_SOVERSION="1.6"
+
+RDEPEND="dev-libs/libpcre
+	dev-libs/zziplib
+	media-libs/lib3ds
+	sys-libs/zlib
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2 )"
+DEPEND="${RDEPEND}"
+
+# This is needed or you get an error on install
+BUILD_DIR="${S}"/build
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0_p864-expat.patch
+	"${FILESDIR}"/${PN}-1.2.2-soversion.patch
+	"${FILESDIR}"/${PN}-1.2.2-no-undefined.patch
+	"${FILESDIR}"/${PN}-1.2.2-libdir.patch
+)
+
+src_prepare() {
+	edos2unix CMakeLists.txt
```

Comment:
Strange. edos2unix runs on my system without that eclass. The patch would fail otherwise. Is it part of EAPI6 then since it just works?